fastentrypoints.py

```diff
@@ -82,21 +82,21 @@ def get_args(cls, dist, header=None):  # noqa: D205,D400
 
 
 def main():
-    import os
     import re
     import shutil
     import sys
+    from pathlib import Path
```

Comment:
Suggestion: Change this upstream. We can get it later along with any other changes.